[perfmon] Fix typo in variable name

gthvn1 opened the pull request.

last-genius commented on python3/bin/perfmon:
```diff
@@ -944,7 +944,7 @@ class SRMonitor(ObjectMonitor):
             & 'sum' for everything else)
```
might as well change it above on L943

gthvn1 commented:
after installing opentelemetry-api and opentelemetry-sdk I still have some issues:
```
>       assert msg[4].startswith("tracers=[<opentelemetry.sdk.trace.Tracer object at")
E       AssertionError: assert False
E        +  where False = <built-in method startswith of str object at 0x7f4128b5bad0>('tracers=[<opentelemetry.sdk.trace.Tracer object at')
E        +    where <built-in method startswith of str object at 0x7f4128b5bad0> = 'Overriding of current TracerProvider is not allowed'.startswith

tests/observer/it_traces.py:104: AssertionError
----------------------------------------------------------------------------------------------- Captured log call ------------------------------------------------------------------------------------------------
DEBUG    __main__:observer.py:400 configs = ['/home/gthouvenin/git/xen-api/python3/tests/observer/observer.conf']
DEBUG    __main__:observer.py:95 /home/gthouvenin/git/xen-api/python3/tests/observer/all.conf: {'module_names': 'XenAPI,tests.observer.traced_script'}
DEBUG    __main__:observer.py:155 module_names: ['XenAPI', 'tests.observer.traced_script']
DEBUG    __main__:observer.py:95 /home/gthouvenin/git/xen-api/python3/tests/observer/observer.conf: {'otel_resource_attributes': '"key1=value1,key2=value2"'}
WARNING  opentelemetry.trace:__init__.py:537 Overriding of current TracerProvider is not allowed
DEBUG    __main__:observer.py:267 tracers=[<opentelemetry.sdk.trace.Tracer object at 0x7f4126131d90>]
============================================================================================ short test summary info =============================================================================================
FAILED tests/observer/it_traces.py::it_creates_a_tracer - AssertionError: assert False
========================================================================================= 1 failed, 109 passed in 0.40s ==========================================================================================
```
Actually, my point was that on my laptop, all tests related to `perfmon` pass. I can try, but I think that if I just keep the comment in my PR, the CI tests will fail as well.

gthvn1 commented:
I think the issue is that I added a line that isn't tested. So, if I understand correctly, it is the coverage of the newly added code that is checked. In my case, 100% of what I added is not tested. I will see how I can write a test for this line.

psafont commented:
Yes a test got added that covers some the changed lines. Thanks, Guillaume!
